pyseir/inference/infer_rt.py

```diff
@@ -80,21 +80,25 @@ def __init__(self,
                 load_data.load_hospitalization_data_by_state(self.state_obj.abbr, t0=self.ref_date)
             self.display_name = self.state
         else:
-            self.agg_level = AggregationLevel.COUNTY
-            self.geo_metadata = load_data.load_county_metadata().set_index('fips').loc[fips].to_dict()
-            self.state = self.geo_metadata['state']
-            self.state_obj = us.states.lookup(self.state)
-            self.county = self.geo_metadata['county']
-            if self.county:
-                self.display_name = self.county + ', ' + self.state
-            else:
-                self.display_name = self.state
-
-            # TODO Swap for new data source.
-            self.times, self.observed_new_cases, self.observed_new_deaths = \
-                load_data.load_new_case_data_by_fips(self.fips, t0=self.ref_date)
-            self.hospital_times, self.hospitalizations, self.hospitalization_data_type = \
-                load_data.load_hospitalization_data(self.fips, t0=self.ref_date)
+            try:
+                self.agg_level = AggregationLevel.COUNTY
+                self.geo_metadata = load_data.load_county_metadata().set_index('fips').loc[fips].to_dict()
+                self.state = self.geo_metadata['state']
+                self.state_obj = us.states.lookup(self.state)
+                self.county = self.geo_metadata['county']
+                if self.county:
+                    self.display_name = self.county + ', ' + self.state
+                else:
+                    self.display_name = self.state
+
+                # TODO Swap for new data source.
+                self.times, self.observed_new_cases, self.observed_new_deaths = \
+                    load_data.load_new_case_data_by_fips(self.fips, t0=self.ref_date)
+                self.hospital_times, self.hospitalizations, self.hospitalization_data_type = \
+                    load_data.load_hospitalization_data(self.fips, t0=self.ref_date)
+            except Exception:
+                logging.warn(f'Failed to init inference for {self.display_name}')
```

Comment:
This happens a lot, and it's not clear why.  there probably more performance improvements simply because fips is being passed through this unnecessarily

Reply:
I guess I'm not sure that catching the exception here is the right place for it - we'll be failing silently (I think `return None` is a no-op) and I'm not sure what actually causes the failure (all of this code looks relatively benign).  

My only thought is that if it's spawning a ton of processes and everyone's trying to run `load_data.load_county_metadata()` for the first time maybe it's an issue? 

I guess I would much rather catch the exception in the calling code because I don't deterministically know what's going to happen here.

Reply:
it's happening because the wrong fips are sent during the county fips generation, either cause the data isn't sufficient, or the fips whitelisted at earlier step but not filtered at a subsequent step.

Reply:
I also don't like this switch. It could be indicating problems like the leading zeros one.  Can we undo and just solve the underlying problem?

Reply:
Still would like to not catch the exception here.  At the very least, can we wrap where this exception is being raised in a try catch and add sentry reporting?